### PR TITLE
Static provisioning for powermax

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
Added example for static provisioning of PowerMax volumes

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/169 |

# Checklist:

- [X] Have you run grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [X] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by boyamurthy — DISMISSED** [review]
looks good to me.

File: content/docs/csidriver/installation/test/powermax.md [hunk 1]
```diff
@@ -70,3 +70,89 @@ Use this procedure to perform a volume expansion test.
...
+    requests:
+      storage: 8Gi
+  storageClassName: powermax
+  volumeMode: Filesystem
+  volumeName: pvol         
+```
+
+4. Then use this PVC as a volume in a pod.
+
+```yaml
```
Comment (coulof) [inline content/docs/csidriver/installation/test/powermax.md:122]: Indentation is broken, please fix

**Review by coulof — CHANGES_REQUESTED** [review]

File: content/docs/csidriver/installation/test/powermax.md [hunk 2]
```diff
@@ -70,3 +70,89 @@ Use this procedure to perform a volume expansion test.
   - Writes some data to the PVC
   - After that, it calculates the checksum of the written data, expands the PVC, and then recalculates the checksum
   - Cleans up all the resources that were created as part of the test
+
+## Consuming existing volumes with static provisioning
```
Comment (coulof) [inline content/docs/csidriver/installation/test/powermax.md:74]: Does this procedure work with any masking view name ? Or will the driver figure out and will create the necessary objects.

The request was from a customer who wanted to pre-provision a LUN.

**Comment by prablr79** [comment]
@shanmydell  why this PR still not addressed ? treat this priority.

File: content/docs/csidriver/installation/test/powermax.md (hunk 2 above)
Comment (boyamurthy) [reply content/docs/csidriver/installation/test/powermax.md:74]: hi @coulof  , the driver will create the necessary objects like SG , Host id and MV .

File: content/docs/csidriver/installation/test/powermax.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/test/powermax.md:122]: @coulof : Yes

Comment (coulof) [reply content/docs/csidriver/installation/test/powermax.md:122]: indentation is still broken

File: content/docs/csidriver/installation/test/powermax.md (hunk 2 above)
Comment (coulof) [reply content/docs/csidriver/installation/test/powermax.md:74]: So a customer is forbidden to create these objects right ?
If so let's add a note to say the same.

File: content/docs/csidriver/installation/test/powermax.md [hunk 3]
```diff
@@ -70,3 +70,89 @@ Use this procedure to perform a volume expansion test.
...
+```
+
+4. Then use this PVC as a volume in a pod.
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: powermaxtest
+    namespace: test
```
Comment (coulof) [inline content/docs/csidriver/installation/test/powermax.md:127]: Sorry to be picky, but above you use 2 spaces for indentation and here 4 spaces.
I know the result is the same, but for sake of readability can we be consistent.

Thx

File: content/docs/csidriver/installation/test/powermax.md (hunk 2 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/test/powermax.md:74]: Addressed

File: content/docs/csidriver/installation/test/powermax.md (hunk 3 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/test/powermax.md]: Addressed

**Review by coulof — APPROVED** [review]

**Review by delldubey — APPROVED** [review]
lgtm

**Pull request merged**